qcfractal/storage_sockets/sqlalchemy_socket.py

```diff
@@ -1413,17 +1413,20 @@ def get_wavefunction_store(
         ----------
         id : List[str], optional
             A list of ids to query
-        include : Dict[str, bool], optional
-            Description
+        include : list/set/tuple of str, default is None
```

Comment:
As with all typed python, these types should be identical to the python types. So `Optional[List[str]]` etc.